hoot-core/src/main/cpp/hoot/core/io/OsmJsonReader.cpp

```diff
@@ -363,8 +359,14 @@ void OsmJsonReader::_parseOverpassNode(const pt::ptree& item)
 
   if (_nodeIdMap.contains(id))
   {
-    throw HootException(
-      QString("Duplicate node id %1 in map %2 encountered.").arg(id).arg(_path));
+    if (_ignoreDuplicates)
```

Comment:
With ignore duplicates flag turned on they are just ignored.  This is important because when requesting data in a bounding box, some ways may extend past the bbox in order for completeness.  That means we can request two neighboring bboxes that will contain the same way that stretches between the two.  In this case a duplicate is expected and one can safely be ignored.  This is applied to nodes, ways, and relations.